src/Footer.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components/macro";
+
+const Blur = styled.div`
+  position: relative;
+  bottom: 0;
+  z-index: 99;
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  backdrop-filter: blur(2.5rem);
+`;
+
+export const Footer = (): JSX.Element => {
+  return (
+    <Blur>
+      <span>
+        Background photo by{" "}
```

Comment:
```suggestion
        Background photo by
```
You can remove all the {" "}

Reply:
I need these for spaces between the span and anchor tags 👀